rlbot_gui/bot_management/downloader.py

```diff
@@ -1,16 +1,31 @@
+import json
 import os
 import tempfile
+import time
 import urllib.request
 import zipfile
-import json
-import eel
-import time
+import multiprocessing as mp
+from functools import partial
 from pathlib import Path
 from shutil import rmtree
 from typing import Optional
 
+import eel
+from rlbot_gui.persistence.settings import load_settings
+
+RELEASE_TAG = 'latest_botpack_release_tag'
+
 
-def download_and_extract_zip(download_url: str, local_folder_path: Path,
+def remove_empty_folders(root: Path):
+    for path, _, _ in list(os.walk(root))[::-1]:
```

Comment:
What is the `[::-1]` doing in this context? Are you skipping a particular value from os.walk? Please document in code and provide a stack overflow link or something if applicable.